### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cru-Luis-Rodriguez**

### CODE REVIEW

**Comment by cru-Luis-Rodriguez** [comment]
@wrandall22 Let me know what else needs to be modified.

File: pom.xml [hunk 1]
```diff
@@ -25,11 +25,7 @@
     <description>aws-sdk</description>
 
     <modules>
-        <module>core</module>
```
Comment (wrandall22) [inline pom.xml]: You should delete the files/folders while you're at it for all of these modules.

**Comment by wrandall22** [comment]
You don't need any of the files/folders besides `install` under `ui.apps`.

**Comment by wrandall22** [comment]
Still need the `aws-java-sdk-osgi-1.11.534.jar` file.

**Comment by cru-Luis-Rodriguez** [comment]
@wrandall22 test build this package when you get a chance -- 
Thanks.

**Review by wrandall22 — APPROVED** [review]

**Pull request merged**